src/renderer/components/ContextMenu/NFTContextMenu.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from "react";
+import { useTranslation } from "react-i18next";
+import IconOpensea from "~/renderer/icons/Opensea";
+import IconRarible from "~/renderer/icons/Rarible";
+import IconGlobe from "~/renderer/icons/Globe";
+import { openURL } from "~/renderer/linking";
+import ContextMenuItem from "./ContextMenuItem";
+
+type Props = {
+  contract: string,
+  tokenId: string,
+  leftClick?: boolean,
+  children: any,
+};
+
+export default function NFTContextMenu({ leftClick, children, contract, tokenId }: Props) {
+  const { t } = useTranslation();
+
+  const menuItems = [
+    {
+      key: "opensea",
+      label: t("NFT.viewer.actions.open", { viewer: "Opensea.io" }),
+      Icon: IconOpensea,
+      type: "external",
+      callback: () => openURL(`https://opensea.io/assets/${contract}/${tokenId}`),
```

Comment:
would be best to move these urls into live-common as env variables i think :thinking:

Reply:
Not sure if we need it as an env or not, but it doesn't harm